timescaledb-restore/README.md

```diff
@@ -0,0 +1,27 @@
+# Restore database from backups
+
+Below is instructions how to spin up CCD scan database locally which is restored from Stage- or Testnet backups.
+
+Before following below instructions make sure you have read access to the S3 buckets where the backups are located.
+
+## Database restore
+
+Build custom docker image by running this
+```
+docker build -t timescale-restore:1 .
+```
+
+Make a `pgbackrest.conf` file like `pgbackrest.conf.example` replacing `$AWS_ACCESS_KEY` and `$AWS_SECRET_KEY` with your access credentials.
+
+Start docker-compose
+```
+docker-compose up
+```
+
+Database is now available at `localhost:15432` with user `postgres` and password `password`.
+
+One can start a timescale database without restoring by explicit in 'docker-compose.yaml' set command to entrypoint to nothing, 'command: [""]'. Default value is 'restore'.
```

Comment:
Wouldn't it be better if the default is not to restore? But I would also prefer setting an env var such that you don't need to edit `docker-compose.yaml`.

Reply:
The whole setup is about having it restore, so IMHO restore is what would be the expected default.

By having it as a environment variable I would interpret it more as a configurations. Changing the command however changes how to whole application work (or you can say don’t do anything). That to me is fair to have as a command and ok the user needs to do some.

Hence I don’t agree on this :-)